### PR TITLE
Fix couchdb seeding script

### PR DESCRIPTION
**Pull request by christulin**
## Purpose of Pull Request
Nest 5 different photos for each room.. generate 10M rooms

### CODE REVIEW

File: database/couchdb.js [hunk 1]
```diff
@@ -20,7 +20,7 @@ const seed = async (database) => {
           const caption = faker.commerce.productName();
           const is_primary = true;
           roomPhotos.push({ storage_url, name, caption, is_primary });
-          const room_number = (run + 1) * i;
+          const room_number = (run + i + j);
           docs.push({room_number, photos: roomPhotos});
```
Comment (amosNash) [inline database/couchdb.js:24]: This looks good. Each room will have 5 photos associated with it?

File: database/couchdb.js [hunk 2]
```diff
@@ -10,7 +10,7 @@ const seed = async (database) => {
   const photos = await listPhotos();
   let docs = [];
 
-  for (let run = 0; run < 1000; run++) {
+  for (let run = 0; run < 10000; run++) {
```
Comment (amosNash) [inline database/couchdb.js:13]: Doesn't this need to be 10M? Or were you just making sure it ran?

**Review by amosNash — COMMENTED** [review]
Seeding script looks good!

File: database/couchdb.js (hunk 1 above)
Comment (christulin) [reply database/couchdb.js:24]: yes, 5 photos with each room and one marked as primary

File: database/couchdb.js (hunk 2 above)
Comment (christulin) [reply database/couchdb.js:13]: there are a 1000 different photos in s3 bucket.. this script amounts to 10M records in DB.. 1000 for this outer loop only amounted in 1M

**Pull request merged**